l10n_br_stock_account/report/stock_valuation_history_xls.py

```diff
@@ -0,0 +1,163 @@
+            'quantity': {
+                'header': [1, 15, 'text', _render("('Quantidade')")],
+                'lines': [1, 0, 'number', _render("line.get('quantity', 0)")],
+                'totals': [1, 0, 'text', None]},
+            'inventory_value': {
+                'header': [1, 15, 'text', _render("('Valor Total')")],
+                'lines': [1, 0, 'number',
+                          _render("line.get('inventory_value', 0)"),
+                          None, self.line_cell_style_decimal],
+                'totals': [1, 0, 'text', None]},
+        }
+
+    def generate_xls_report(self, _p, _xs, data, objects, wb):
+
+        report_name = 'Relatório de Valorização de Estoque'
+        ws = wb.add_sheet(report_name[:31])
+        ws.panes_frozen = True
+        ws.remove_splits = True
+        ws.portrait = 0  # Landscape
+        ws.fit_width_to_pages = 1
+        row_pos = 0
+
+        # set print header/footer
+        ws.header_str = self.xls_headers['standard']
+        ws.footer_str = self.xls_footers['standard']
+
+        # Title
+        c_specs = [
+            ('report_name', 1, 0, 'text', report_name),
+        ]
+        row_data = self.xls_row_template(c_specs, ['report_name'])
+        row_pos = self.xls_write_row(
+            ws, row_pos, row_data, row_style=xlwt.easyxf(_xs['xls_title']))
+        row_pos += 1
+
+        # Column headers
+        c_specs = map(lambda x: self.render(
+            x, self.col_specs_template, 'header'),
+            [
+                'product_id',
+                'fiscal_classification_id',
+                # 'product_id_count',
+                'quantity',
+                'price_unit_on_quant',
+                'inventory_value'
+            ])
+        row_data = self.xls_row_template(c_specs, [x[0] for x in c_specs])
+        row_pos = self.xls_write_row(
+            ws, row_pos, row_data, row_style=self.rh_cell_style_center,
+            set_column_size=True)
+        ws.set_horz_split_pos(row_pos)
+
+        # Lines
+        active_ids = self.context.get('active_ids')
+        wizard_obj = self.pool.get('wizard.valuation.history').browse(
+            self.cr, self.uid, active_ids)
+        data = wizard_obj.compute(
+            self.cr, self.uid, objects[0].id, date=wizard_obj.date,
+            context=self.context)
+        for line in data:
+            c_specs = map(
+                lambda x: self.render(
+                    x, self.col_specs_template, 'lines'), [
+                    'product_id',
+                    'fiscal_classification_id',
+                    # 'product_id_count',
+                    'quantity',
+                    'price_unit_on_quant',
+                    'inventory_value'
+                ])
+            row_data = self.xls_row_template(c_specs, [x[0] for x in c_specs])
+            row_pos = self.xls_write_row(
+                ws, row_pos, row_data, row_style=self.line_cell_style)
+        pass
```

Comment:
Isn't this pass statement useless? Looks like something left from an old prototype may be? Or am I wrong?